docs/UserGuide.md

```diff
@@ -368,7 +386,7 @@ Additionally, if you would like to delete a tag and all its sub-tags, you can sp
 Format: `delete -t t/TAG_NAME [r/BOOLEAN]`
 
 * `TAG_NAME` must be the name of an existing tag in Athena.
-* `BOOLEAN` must be of the form `1`, `t`, `true` for true or `0`, `f`, `false` for false.
+* `BOOLEAN` must be of the form `1`, `t`, `true` for a true case or `0`, `f`, `false` for a false case.
```

Comment:
sentences' connotations are the same.

Reply:
In markdown, the difference is significantly more prominent. I do not think this is a necessary change.

Reply:
I feel that it is more of a grammatical error, hence the sentence connotation will definitely be the same still.

Reply:
This is a user guide, and the original line is in a coders language. I think this change makes it easier to understand for a non coder and is a useful change.